rerun_py/rerun_sdk/rerun2/components/color_ext.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import annotations
+
+__all__ = ["ColorArrayExt"]
+
+from typing import Any, Sequence, Type
+
+import numpy as np
+import pyarrow as pa
+
+from rerun.color_conversion import u8_array_to_rgba
+
+
+class ColorArrayExt:
+    @staticmethod
+    def _from_similar(data: Any | None, *, mono: type, mono_aliases: Type, many: type, many_aliases: Type, arrow: type):
+        """
+        Normalize flexible colors arrays.
+
+        Float colors are assumed to be in 0-1 gamma sRGB space.
+        All other colors are assumed to be in 0-255 gamma sRGB space.
+
+        If there is an alpha, we assume it is in linear space, and separate (NOT pre-multiplied).
+        """
```

Comment:
We need to surface this comment somehow to the user where appropriate. Here it won't be all that useful

Reply:
I'll add this blurb as tagged docs.

Though again, this is just an alpha preview, we'll give the doc much more love when backporting.